atc/step_validator.go

```diff
@@ -81,19 +74,12 @@ func (validator *StepValidator) VisitTask(plan *TaskStep) error {
 	}
 
 	if plan.Config != nil && (plan.Config.RootfsURI != "" || plan.Config.ImageResource != nil) && plan.ImageArtifactName != "" {
-		validator.recordWarning(ConfigWarning{
+		validator.recordConfigErrors(ConfigErrors{
 			Type:    "pipeline",
 			Message: validator.annotate("specifies image: on the step but also specifies an image under config: - the image: on the step takes precedence"),
 		})
 	}
 
-	if plan.Hermetic {
-		validator.recordWarning(ConfigWarning{
-			Type:    "pipeline",
-			Message: validator.annotate("specifies `hermetic:` only works against worker containerd runtime"),
-		})
-	}
```

Comment:
oh this is unfortunate, but I don't think we should keep the warning stuff just for this one warning. We warn about this in the docs already.
I'm mostly commenting so folks know I saw this and accepted that we won't issue this warning for tasks that set `hermetic` anymore.